plugins/lsp/language-servers/default.nix

```diff
@@ -210,6 +210,15 @@ with lib; let
       package = pkgs.elixir-ls;
       cmd = cfg: ["${cfg.package}/bin/elixir-ls"];
     }
+    {
+      name = "fortls";
+      cmd = cfg: [
+        "${cfg.package}/bin/fortls"
+        "--hover_signature"
+        "--hover_language=fortran"
+        "--use_signature_help"
+      ];
```

Comment:
```suggestion
```

As this is the default `nvim-lspconfig` value for `cmd`, I am not sure that you have to add this...

Reply:
It's not exactly the default command, I remove the startup notification.

Reply:
Ok fine !